lua/nvim-drawer/init.lua

```diff
@@ -524,8 +619,6 @@ function mod.setup(_)
             vim.cmd('qa')
           end
         end
-      else
-        closing_window_instance.close({ save_size = false })
```

Comment:
I got rid of this while debugging some stuff with nvim-tree, should test to see if that's still happening after all the refactoring that went on.